yarn.lock

```diff
@@ -1,5 +1,7 @@
 # THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This changed when i installed the root dependencies. I will revert the changes made to this file.